source/funkin/Conductor.hx

```diff
@@ -484,16 +490,29 @@ class Conductor
       this.onMeasureHit.dispatch();
     }
 
+    this.songPositionDelta += FlxG.elapsed * 1000 * (FlxG.sound?.music?.pitch ?? 1);
```

Comment:
Just realized this increments even when the song is paused.
Maybe move it to line 430?
```haxe
    // If the song is playing, limit the song position to the length of the song or beginning of the song.
    if (FlxG.sound.music != null && FlxG.sound.music.playing)
    {
      this.songPosition = Math.min(currentLength, Math.max(0, songPos));
      this.songPositionDelta += FlxG.elapsed * 1000 * (FlxG.sound?.music?.pitch ?? 1);
    }
    else
    {
      this.songPosition = songPos;
    }
```